pkg/check/util.go

```diff
@@ -227,3 +248,24 @@ func vCentersToMap(vcenters []ocpv1.VSpherePlatformVCenterSpec) map[string]ocpv1
 	}
 	return vcenterMap
 }
+
+func GetVCenter(checkContext *CheckContext, node *v1.Node) *VCenter {
+	// Determine which vCenter connection is in by looking at failure domain info
+	// failure-domain.beta.kubernetes.io/region: us-east
+	// failure-domain.beta.kubernetes.io/zone: us-east-4a
+	region := node.Labels["failure-domain.beta.kubernetes.io/region"]
+	zone := node.Labels["failure-domain.beta.kubernetes.io/zone"]
+
+	server := ""
+	// Get failure domain
+	for _, fd := range checkContext.PlatformSpec.FailureDomains {
+		if fd.Region == region && fd.Zone == zone {
+			server = fd.Server
+			break
+		}
+	}
+
+	vCenterinfo := checkContext.VCenters[server]
```

Comment:
potential for unknown vCenter

Reply:
I think I changed this around when you reviewed.  I am not seeing this code statement anymore.